src/Dfc.CourseDirectory.WebV2/Services/UkrlpWcfService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Dfc.CourseDirectory.WebV2.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UkrlpService;
+
+namespace Dfc.CourseDirectory.WebV2.Services
+{
+    public class UkrlpWcfService : IUkrlpWcfService
+    {
+        public async Task<ProviderRecordStructure> GetProviderData(int ukprn)
+        {
+            string[] statusesToFetch =
+            {
+                    "A", // Active
+            };
+
+            List<ProviderRecordStructure> results = new List<ProviderRecordStructure>();
+            foreach (String status in statusesToFetch)
+            {
+                var request = BuildRequest(ukprn.ToString());
+                request.SelectionCriteria.ProviderStatus = status;
+                request.QueryId = GetNextQueryId();
```

Comment:
Is this required? `GetNextQueryId` looks like it will always return the same ID...